website/content/docs/concepts/security/permissions/grants.mdx

```diff
@@ -0,0 +1,182 @@
+---
+layout: docs
+page_title: Grant Strings
+description: |-
+  Grant Strings
+---
+
+# Grant Strings
+
+In Boundary, grants are a set of permissions that you give to the principals, or
+[users](/boundary/docs/concepts/domain-model/users) and [groups](/boundary/docs/concepts/domain-model/groups),
+of a specific role defined in a specific scope.
+
+When you write grant strings, you should consider the following questions:
+
+- Which specific scope should these permissions be applied to?
+- Which specific role should have these permissions?
+- Which specific resources should this grant apply to?
+- What resource types should this apply to?
+- What kinds of actions should the principals (users and groups) be able to perform?
+
+Grant strings can be composed of up to 4 parts:
+- [IDs](#ids)
+- [Type](#type)
+- [Actions](#actions)
+- [Output fields](#output-fields)
+
+You use selectors to indicate which resources the grant should apply to.
+Selectors can be specific IDs or wildcard IDs and type selectors. There are currently two types of selectors:
+
+- The `ids` field indicates specific resources or a wildcard to match all
+
+- A `type` field that indicates a specific resource type or a wildcard to match
+  all; it might also be used to grant permissions on collections of resources
+
+Additionally, there are two types of assigned permissions:
+
+- An `actions` field indicates which actions to allow the client to perform on
+  the resources matched by `ids` and `type`
+
+- An `output_fields` field indicates which top-level fields to return in the
+  response
+
+The format of a full grant string is:
+
+- &#32; `ids=<ids>;type=<type>;actions=<action-list>;output_fields=<fields-list>`
+
+You can find grant [examples](#common-grant-examples) at the bottom of this page.
+
+<Note>
+`output_fields` is omitted in most examples for brevity, but they are valid
+in all of them. It is also valid in each case to omit `actions` and specify
+_only_ `output_fields`.
+</Note>
```

Comment:
```suggestion
<Note>

`output_fields` is omitted in most examples for brevity, but they are valid
in all of them. It is also valid in each case to omit `actions` and specify
_only_ `output_fields`.

</Note>
```

Reply:
The `monospace code` font wasn't showing up accurately in the Note above. I think you need to add a blank line between each of the `Note` tags and the paragraph.